docs/user/discover.asciidoc

```diff
@@ -283,3 +283,5 @@ include::{kib-repo-dir}/discover/search-for-relevance.asciidoc[]
 include::{kib-repo-dir}/discover/save-search.asciidoc[]
 
 include::{kib-repo-dir}/discover/search-sessions.asciidoc[]
+
+include::{kib-repo-dir}/discover/document-explorer.asciidoc[]
```

Comment:
Is this in the right place in the TOC? I would think this content is similar to `View a document`, so maybe bump it up?